server/container/guice/mailbox-plugin-deleted-messages-vault-guice/pom.xml

```diff
@@ -37,6 +37,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-deleted-messages-vault</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-deleted-messages-vault-cassandra</artifactId>
```

Comment:
does that mean that you require cassandra for the deleted messages vault plugin?
Usually, we create a separated guice project for backends

Reply:
the splitting in later commit resolved this concern